crosswalk/index.html

```diff
@@ -279,25 +280,25 @@ <h2>EPUB and ONIX</h2>
 						</td>
 					</tr>
                     
-					<tr>
+					<tr id="eaa-microenterprise">
```

Comment:
We don't need IDs here, although maybe they don't hurt anything. It was the note that we needed to deep link into and that's done now. For this document, it's the hyperlinks that need to be fixed to be more specific. I'll suggest the changes for each. I'll create suggestions to remove the IDs but feel free to ignore the suggestions if you want to keep them.